<?xml version='1.0' ?>

<launch>

  <arg name="use_sim_time" default="true" description="Use the /clock topic for time to sync with simulation"/>
  <arg name="manager_address" default="localhost" description="The address for the manager"/>
  <arg name="manager_port" default="5003" description="The port for the manager"/>
  <arg name="output" default="screen"/>

  <!-- Lift manager -->
  <node pkg="rmf_demos_lift_adapter" exec="lift_manager" output="both">
    <param name="manager_address" value="$(var manager_address)"/>
    <param name="manager_port" value="$(var manager_port)"/>
    <param name="use_sim_time" value="$(var use_sim_time)"/>
  </node>

  <!-- Lift adapter -->
  <node pkg="rmf_demos_lift_adapter" exec="lift_adapter" output="both">
    <param name="manager_address" value="$(var manager_address)"/>
    <param name="manager_port" value="$(var manager_port)"/>
    <param name="use_sim_time" value="$(var use_sim_time)"/>
  </node>

</launch>
